docs/integrations/sources/surveymonkey.md

```diff
@@ -1,48 +1,64 @@
 # SurveyMonkey
 
-This page guides you through the process of setting up the SurveyMonkey source connector.
+Learn how to set up the SurveyMonkey source connector. A source connector extracts data from SurveyMonkey and transfers the data to a destination. Learn more about connectors [here](/using-airbyte/core-concepts/).
 
 :::note
 
-OAuth for Survey Monkey is officially supported only for the US. We are testing how to enable it in the EU at the moment. If you run into any issues, please [reach out to us](mailto:product@airbyte.io) so we can promptly assist you.
+OAuth for SurveyMonkey is available in the US only. We are testing how to enable OAuth in the EU. [Reach out to us](mailto:product@airbyte.io) if you run into any issues with authentication.
```

Comment:
1. Removed unnecessary modifiers such as "officially" and "promptly." These words don't add much to the description and clutter a page. 

2. Also, a common mistake in tech writing is over-clarifying something. When we say OAuth is available in the US only, we're already implying the "officially" part and can leave that word out.

3. I decided to add "with authentication" at the end. Because the topic is OAuth, the issue should be related to OAuth, and we can specify the type of issue.